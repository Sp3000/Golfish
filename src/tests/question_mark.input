51?n60?n~7"a"?n8""?n~95?Snan;