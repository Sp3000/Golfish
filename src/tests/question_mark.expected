5 7 910